src/Troubleshooting/index.md

```diff
@@ -8,6 +8,79 @@ This is due to a Roblox update causing an incompatibility with Wayland's handlin
+This may also indicate your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
+
+#### Make sure there are no stale wineservers running
+
+There are multiple reasons for why this would occur, but `vinegar kill` will suffice in order to fix the issue.
+
+#### Roblox Studio stuck on the splash screen
+
+This usually happens due to issues with the DPI. In order to fix it, set the DPI to 97 by running `vinegar exec winecfg` to open the Wine configuration, and then head to the Graphics tab. There should be a DPI option at the bottom of the page.
+
+A DPI greater than 96 could be set, but this causes in-game GUIs to be larger than normal. Also, because of a Wine issue, using a non-default DPI will cause the Roblox viewport to be a bit blurry. Additionally, make sure that the DPI is an even number, otherwise GUIs will be blurry.
+
+#### Unable to log in through the Studio or Player apps
+
+This occurs due to WebView2 issues with Wine.
+
+There are 2 workarounds:
+
+- Quick Login (Player): Click on "Log in with another device", and enter the code displayed on the other device/website.
+  
+- Website (Player and Studio): Log in through the [Roblox website](https://roblox.com) and launch/edit an experience.
+
+#### "Your GPU is incompatible" error when launching Roblox Studio
+
+This issue is most commonly caused by the Wine build not having the Vulkan ChildWindow patch applied. To fix it:
+
+- Use a Wine build that has the ChildWindow patch enabled (such as [Wine GE](https://github.com/GloriousEggroll/wine-ge-custom)), or:
+
+- Disable DXVK in the Vinegar configuration.
+
+#### Memory Dump
+
+This is because of [commit ea640f6c](https://gitlab.winehq.org/wine/wine/-/commit/ea640f6cece7660ffc853b7d574fbe52af34901a) (on Wine 8.16 and newer) causing Roblox to create a memory dump.
+
+There are 2 workarounds:
+
+- Using the [Vinegar flatpak](https://vinegarhq.github.io/Installation/guides/flatpak.html).
+
+- Using [Wine-GE](https://github.com/GloriousEggroll/wine-ge-custom/releases) or a patched build of Wine with the above commit reverted.
+
+#### Buggy Roblox Studio fonts
+
+This is due to an incompatibility with WineD3D. Enabling DXVK in the configuration file should solve this issue.
+
+#### Vulkan/Vulkan.h not found when compiling Vinegar
+
+Vulkan development tools are required for building Vinegar with splash screen support. They're usually installed using a separate package, such as: 
+- `vulkan-loader-dev` in Ubuntu.
+- `vulkan-loader-devel` in Fedora.
+- `vulkan-devel` in Arch-based distros.
+
+  
+distribution that doesn't ship the development packages/tools.
+If using a distribution that doesn't ship the development packages, they may be compiled using the following guide: [Vulkan-Loader Build Instructions](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/BUILD.md#building-on-linux).
```

Comment:
@tacheometry Hmm, I know you dislike personal pronouns, but I think this phrase is really needing a "you're". It sounds weird otherwise. Maybe it should just be rewritten altogether?

Reply:
It's fine with the "you're"